orc8r/cloud/go/services/certifier/protos/utils.go

```diff
@@ -0,0 +1,51 @@
+package protos
+
+import (
+	"github.com/golang/protobuf/proto"
+
+	"magma/orc8r/cloud/go/blobstore"
+)
+
+const (
+	// UserType is the type of CertInfo used in blobstore type fields.
+	UserType = "user"
+	// PolicyType is the type of policy used in blobstore type fileds
+	PolicyType = "policy"
+)
+
+func UserFromBlob(blob blobstore.Blob) (User, error) {
```

Comment:
These 4 are really just "proto to/from blob" with passed type+key. I'm fine with keeping the definitions as-is, but at minimum let's use a shared impl

Reply:
P.S. 2 things

1. "utils.go" is generally bad practice for a file name. "Helpers" is also bad, but in this case let's use helpers.go since we're adding our own helpers to a generated directory
2. Consider making these methods on the underlying proto types, rather than standalone funcs

Reply:
Cool, made the methods on the underlying proto types... 

How can we have a shared impl when they all have diff types and keys? this would be so nice if golang had generics, but idk how we would otherwise do this?